apps/notepad/src/app/notes.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class NotesService {
+  constructor() {}
+  fetchNotes() {
```

Comment:
fetchNotes function return value implicitly has type any. A more specific type can be used.